Add basic example site (#653)
scientific-python/scientific-python-hugo-theme: doc/content/user_guide/getstarted.md

> 3. Copy the theme's documentation site as a template for your site:

Reviewer: ```suggestion
3. Copy the theme's example site as a template:
```
